pyproject.toml

```diff
@@ -1,3 +1,21 @@
+[project]
+name = "openmethane_prior"
+version = "0.0.1"
+authors = [
+  { name="Placeholder Author", email="placeholder@example.com" },
```

Comment:
Can you make an issue asking Peter who the authors of this package should be. I would add you to that list now

Reply:
```suggestion
  { name="Peter Rayner", email="peter.rayner@superpowerinstitute.com.au" },
  { name="Gerard Mason", email="gerard.mason@superpowerinstitute.com.au" },
  { name="Jared Lewis", email="jared.lewis@climate-resource.com" },
  { name="Daniel Busch", email="daniel.busch@climate-resource.com" },
```